src/Files.Uwp/Converters/HasTextConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Files.Converters
```

Comment:
Files.Uwp.Converters

and please change other namespaces too :)

Reply:
Ok, I'll do that.